packages/flutter_tools/lib/src/build_system/targets/icon_tree_shaker.dart

```diff
@@ -119,12 +127,14 @@ class IconTreeShaker {
     );
 
     if (fonts.length != iconData.length) {
-      throwToolExit('Expected to find fonts for ${iconData.keys}, but found '
-                    '${fonts.keys}. This usually means you are refering to '
-                    'font families in an IconData class but not including them '
-                    'in the assets section of your pubspec.yaml, are missing '
-                    'the package that would include them, or are missing '
-                    '"uses-material-design: true".');
+      _logger.printError(
```

Comment:
I'm not sure if this breaks any invariants. It might also need to be a trace event